app/settings.py

```diff
@@ -149,10 +149,14 @@ def utcoffset_or_fail(date_value, key):
     "ACCOUNT_SERVICE_BASE_URL", "https://surveys.ons.gov.uk"
 )
 
-ACCOUNT_SERVICE_BASE_URL_SOCIAL = "https://start.surveys.ons.gov.uk"
+ACCOUNT_SERVICE_BASE_URL_SOCIAL = os.getenv(
+    "ACCOUNT_SERVICE_BASE_URL_SOCIAL", "https://start.surveys.ons.gov.uk"
+)
 
 PRINT_STYLE_SHEET_FILE_PATH = os.getenv(
     "PRINT_STYLE_SHEET_FILEPATH", "templates/assets/styles"
 )
 
 ONS_URL = os.getenv("ONS_URL", "https://www.ons.gov.uk")
+
+CY_ONS_URL = os.getenv("CY_ONS_URL", "https://cy.ons.gov.uk")
```

Comment:
Best to have the same prefix. Census ones are suffix based, but should be prefix.

```suggestion
ONS_URL_CY = os.getenv("ONS_URL_CY", "https://cy.ons.gov.uk")
```